packages/eslint-scope/package.json

```diff
@@ -30,8 +30,6 @@
   "license": "BSD-2-Clause",
   "scripts": {
     "build": "rollup -c",
-    "build:update-version": "node tools/update-version.js",
-    "prepublishOnly": "npm run build:update-version && npm run build",
```

Comment:
We're already running `build` before publishing:

https://github.com/eslint/js/blob/04247cbf4b4b99a8eb5389952df0af504814582d/.github/workflows/release-please.yml#L44-L47